cni-plugins/sriov-passthrough-cni/README.md

```diff
@@ -53,9 +53,11 @@ limits:
 
 * In case 2 PFs are required, please do the following changes:
 
-1. Use the following annotation instead those in the previous [1] section.
+1. Use one of the following annotations instead those in the previous [1] section.
 
-    `k8s.v1.cni.cncf.io/networks: '[{"interface":"net1","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}, {"interface":"net2","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}]'`
+    [1] `k8s.v1.cni.cncf.io/networks: 'multus-cni-ns/sriov-passthrough-cni,multus-cni-ns/sriov-passthrough-cni'`
+
+    [2] `k8s.v1.cni.cncf.io/networks: '[{"interface":"net1","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}, {"interface":"net2","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}]'`
 
 2. Follow [2] of the previous section, but instead of 1 `prow/sriov`, use 2, in order to let k8s know that 2 PFs are
```

Comment:
Unrelated, but since you are changing the README, would you please fix this? I believe this should say "Follow [4] ..."

Reply:
meant this [2]
https://github.com/kubevirt/project-infra/pull/1018/files#diff-82391f13a16cc900208dda76428e4ee9c019222c04267e1f6c731859000247a5R39

but will change it
thanks

Reply:
meant this [2] (as it has the whole explanation)
https://github.com/kubevirt/project-infra/pull/1018/files#diff-82391f13a16cc900208dda76428e4ee9c019222c04267e1f6c731859000247a5R39

but will change it in case you still prefer ?

thanks